code/game/objects/effects/poster_motivational.dm

```diff
@@ -0,0 +1,291 @@
+	if(!is_special_character(user))
+		return ..()
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return ..()
+	if(!istype(postertool, /obj/item/toy/crayon))
+		return ..()
+	balloon_alert(user, "converting poster...")
+	if(!do_after(user, 5 SECONDS, user))
+		balloon_alert(user, "interrupted!")
+		return
+	var/obj/item/poster/traitor/quirkspawn = new(get_turf(src))
+	user.put_in_hands(quirkspawn)
+	to_chat(user, span_notice("You have converted one of your posters!"))
+	qdel(src)
+
+/// Screentip for the above
+
+/obj/item/poster/quirk/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	if(!is_special_character(user))
+		return NONE
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return NONE
+	if(!istype(held_item, /obj/item/toy/crayon))
+		return NONE
+	context[SCREENTIP_CONTEXT_LMB] = "Turn into Demoralizing Poster"
+	return CONTEXTUAL_SCREENTIP_SET
+
+/obj/structure/sign/poster/quirk
+	name = "quirk poster"
+	desc = "A large piece of homemade space-resistant printed paper."
+	poster_item_name = "homemade poster"
+	poster_item_desc = "A poster made with love, some people will enjoy seeing it."
+	poster_item_icon_state = "rolled_legit"
+	var/quirk_poster_department = NONE
+	var/datum/proximity_monitor/advanced/quirk_posters/mood_buff
+
+/obj/structure/sign/poster/quirk/on_placed_poster(mob/user)
+	mood_buff = new(src, 7, TRUE, quirk_poster_department)
+	return ..()
+
+/obj/structure/sign/poster/quirk/attackby(obj/item/I, mob/user, params)
+	if (I.tool_behaviour == TOOL_WIRECUTTER)
+		QDEL_NULL(mood_buff)
+	return ..()
+
+/obj/structure/sign/poster/quirk/Destroy()
+	QDEL_NULL(mood_buff)
+	return ..()
+
+/obj/structure/sign/poster/quirk/apply_holiday()
+	var/obj/structure/sign/poster/traitor/holi_data = /obj/structure/sign/poster/quirk/festive
+	name = initial(holi_data.name)
+	desc = initial(holi_data.desc)
+	icon_state = initial(holi_data.icon_state)
+
+/*
+ * Applies a department-based mood if you can see the parent.
+ *
+ * - Applies a mood buff to people who are in visible range of the item.
+ */
+/datum/proximity_monitor/advanced/quirk_posters
+	var/department
+
+/datum/proximity_monitor/advanced/quirk_posters/New(atom/_host, range, _ignore_if_not_on_turf = TRUE, department)
+	. = ..()
+	src.department = department
+	RegisterSignal(host, COMSIG_PARENT_EXAMINE, PROC_REF(on_examine))
+
+/datum/proximity_monitor/advanced/quirk_posters/field_turf_crossed(atom/movable/crossed, turf/location)
+	if (!isliving(crossed))
+		return
+	if (!can_see(crossed, host, current_range))
+		return
+	on_seen(crossed)
+
+/datum/proximity_monitor/advanced/quirk_posters/proc/on_examine(datum/source, mob/examiner)
+	SIGNAL_HANDLER
+	if (isliving(examiner))
+		on_seen(examiner)
+
+/datum/proximity_monitor/advanced/quirk_posters/proc/on_seen(mob/living/viewer)
+	if (!viewer.mind)
+		return
+	if (!viewer.mob_mood)
+		return
+	if (viewer.stat != CONSCIOUS)
+		return
+	if(viewer.is_blind())
+		return
```

Comment:
```suggestion
	if (!viewer.mind || !viewer.mob_mood || (viewer.stat != CONSCIOUS) || viewer.is_blind())
		return
```

Combine the four easy checks into one short-circuited line. Would be nice if you could also fold in the other two below this, but might get too long-winded for one line.